src/views/Setup.vue

```diff
@@ -1,32 +1,31 @@
 <template>
-	<Content app-name="mail">
-		<Navigation v-if="hasAccounts" />
-		<div id="emptycontent">
-			<div class="icon-mail" />
+	<EmptyContent icon="icon-mail">
```

Comment:
I find it odd that you can drop the Content, please double check this is intentional :)

Reply:
Thank you! I have corrected it